### ISSUE TITLE
LISA on Itch.io is a different game than Steam.

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#1179]
Hello.

Itch.io's LISA game is a totally different one from Steam's.

Regards.

[Issue created by [deleted]: 2018-09-23]

**Comment by MarkitkaM** [comment]
Yes, they are different and listed separately actually, not sure how it was before:

**itch.io**
https://isthereanydeal.com/game/lisa1/info/
https://rampagingram.itch.io/lisa
-> Add a thumbnail

**Steam**
https://isthereanydeal.com/game/lisa/info/
https://store.steampowered.com/app/335670/LISA_The_Painful/
